Dockerfile

```diff
@@ -24,9 +24,7 @@ ARG ENABLE_ARCHIVES=true
 ###
 # Set up base stages for building and deploying
 ###
-
-# Get basic configs and Jekyll env
-FROM docs/docker.github.io:docs-builder AS builderbase
+FROM starefossen/github-pages:198 AS builderbase
```

Comment:
This (at least for now) cuts out our custom image, which was based on `starefossen/github-pages`, with some other changes that we currently don't use)

Reply:
Good idea to remove complexity!

Reply:
The idea of the custom image was that we could update or do fixes in that image, and rebuild the archived docs, without having to update each of those branches.

If we stop publishing those archives that may no longer be relevant